README.md

```diff
@@ -68,6 +68,15 @@ Please note that the date shown is the date that the change was released to the
 
 ---
 
+### 15 April 2024
+
+The following changes are now available in the Sandbox:
+
+#### self-employment-business-api
+
+- Added 2 new optional fields transitionProfitAmount, transitionProfitAccelerationAmount in annualAdjustmentsType object for Retrieve a Self-Employment Annual Submission and Create and Amend Self-Employment Annual Submission Endpoints.
```

Comment:
Should probably indicate the version number affected as well as enclosing the endpoint names in back ticks e.g. something like:
```suggestion
- Added 2 new optional fields transitionProfitAmount, transitionProfitAccelerationAmount in annualAdjustmentsType object for the `v3.0` `Retrieve a Self-Employment Annual Submission` and `Create and Amend Self-Employment Annual Submission` endpoints.
```